test/language/statements/class/elements/public-class-field-initialization-on-super-class-with-setter.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2019 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Public class field initialization calls [[DefineOwnProperty]] and don't execute super's getter
+esid: sec-define-field
+info: |
+  DefineField(receiver, fieldRecord)
+    ...
+    8. If fieldName is a Private Name,
+      a. Perform ? PrivateFieldAdd(fieldName, receiver, initValue).
+    9. Else,
+      a. Assert: IsPropertyKey(fieldName) is true.
+      b. Perform ? CreateDataPropertyOrThrow(receiver, fieldName, initValue).
+    10. Return.
+includes: [propertyHelper.js]
+features: [class, class-fields-public]
+---*/
+
+class Super {
+  set f(v) {
+    throw new Test262Error();
```

Comment:
it would be great to have something similar from this and the first file:

```js
  return new Proxy(this, {
      defineProperty: function (target, key, descriptor) {
        throw new Test262Error();
      });
  }

...

assert.throws(Test262Error, () => { new Base(); });
```